HomeWorks/src/ru/geekbrains/evstigneeva/Employee.java

```diff
@@ -0,0 +1,29 @@
+package ru.geekbrains.evstigneeva;
+
+public class Employee {
+
+    String name_fio;
```

Comment:
нижнее подчеркивание не приветствуется в именах полей класса
```suggestion
    String nameFio;
```